tools/convert_bigfile_gadget_format3.py

```diff
@@ -113,7 +116,14 @@ def write_hdf_file(bf, hdf5name, fnum, nfiles):
                 continue
             ptype = int(ptype)
             hname = names.get_hdf5_name(bname)
-            hdf5["PartType"+str(ptype)][hname] =  bf[block][startpart[ptype]:endpart[ptype]]
+            bfdata = bf[block][startpart[ptype]:endpart[ptype]]
+            #Velocities need units adjusted
```

Comment:
# Gadget-3 uses a^{1/2} dx / dt. 

if pecvel: # pecvel is a dx / dt.

else # MP-Gadget@Bluetides snapshot, a^2 dx / dt.